src/github.com/getlantern/flashlight/ui/ui.go

```diff
@@ -97,7 +98,18 @@ func Start(requestedAddr string, allowRemote bool, extUrl string) (string, error
 		}
 		resp.WriteHeader(http.StatusOK)
 	}
+
+	// We use the backend to detect the user's country and redirect the browser
+	// to the correct URL that will itself be proxied over Lantern.
+	feedHandler := func(resp http.ResponseWriter, req *http.Request) {
+		vals := req.URL.Query()
+		defaultLang := vals.Get("lang")
+		url := feed.GetFeedURL(defaultLang)
```

Comment:
It's possible ``defaultLang`` could be missing here, right? If so, ``determineLocale`` doesn't currently account for the empty string case.